cmd/event-notifier_test.go

```diff
@@ -38,7 +39,9 @@ func testEventNotify(obj ObjectLayer, instanceType string, t TestErrHandler) {
 	// remove the root folder after the test ends.
 	defer removeAll(rootPath)
 
-	initEventNotifier(obj)
+	if err := initEventNotifier(obj); err != nil {
+		t.Fatal("Unexpected error:", err)
```

Comment:
We don't need it @harshavardhana, t.Fatal works fine like that
https://golang.org/pkg/testing/#T.Fatal `func (c *T) Fatal(args ...interface{})`

It will just add spaces between arguments